config/install/redirect.settings.yml

```diff
@@ -11,3 +11,4 @@ access_check: false
 normalize_aliases: true
 content_location_header: false
 term_path_handler: true
+error_log: true
```

Comment:
Same for the name, this should be more descriptive. We're not storing errors, we're storing 404 requests.